instruments/tektronix/tektds5xx.py

```diff
@@ -137,7 +135,8 @@ def read_waveform(self, bin_format=True):
                 raw = self._parent.binblockread(data_width)
 
                 # pylint: disable=protected-access
-                self._parent._file.flush_input()  # Flush input buffer
+                # read line separation character
+                self._parent._file.read_raw(1)
```

Comment:
Same issue as with the other Tektronix scopes in the binary data reader. Here again, manual states that the binary form comes with a newline character afterwards, thus replacing the `flush_input()` with `read_raw(1)` should be fair.